rollup.config.js

```diff
@@ -2,21 +2,68 @@ import babel from 'rollup-plugin-babel'
 import css from 'rollup-plugin-css-porter'
 import pkg from './package.json'
 
+const fs = require('fs')
+
+/**
+ * Custom rollup plug-in which bundles all transforms' CSS in a single file.
+ * @param  {callback} options Rollup plug-in options func
+ * @return {void}
+ */
+const wikimediaPageLibraryTransformCSSRollupPlugin = (options = {}) => {
+  const cssBuffer = {}
+  return {
+    transform(code, id) {
+      if (!options.fileSuffixPattern.test(id)) { return }
+      if (!Object.prototype.hasOwnProperty.call(cssBuffer, id) || cssBuffer[id] !== code) {
+        cssBuffer[id] = code
+      }
+      return ''
+    },
+    onwrite(opts) {
+      if (!Object.keys(cssBuffer).length) { return }
+
+      const keys = Object.keys(cssBuffer)
+
+      // We want the theme transform to be first. This is because the theme transform CSS has to use
+      // some '!important' CSS modifiers to reliably set themes on elements which may contain inline
+      // styles. Moving it to the top of the file is necessary so other transforms can override
+      // these '!important' themes transform CSS bits if needed.
```

Comment:
It's a bummer that this complexity has to exist within our own library. The usage of `!important` seems to always end in an arms race that concludes with `!important` inline style or JavaScript. I'm sorry you had to deal with this complexity but I'm glad I was able to learn from it (I certainly wasn't thinking about this when we started) :+1: